our-app/src/components/authenticated/ImmunizationForm.js

```diff
@@ -0,0 +1,278 @@
+                                <Form.Control type='infoStatementDate' placeholder="Info Statement Date">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+
+                    <Row>
+                        <Col>
+                            <Form.Group controlId='lotNum'>
+                                <Form.Label>Lot Num</Form.Label>
+                                <Form.Control type='lotNum' placeholder="Lot Num">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                        <Col>
+                            <Form.Group controlId='mfgName'>
+                                <Form.Label>Mfg Name</Form.Label>
+                                <Form.Control type='mfgName' placeholder="Mfg Name">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+
+                    <Row>
+                        <Col>
+                            <Form.Group controlId='orderingProvider'>
+                                <Form.Label>Ordering Provider</Form.Label>
+                                <Form.Control type='orderingProvider' placeholder="Ordering Provider">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                        <Col>
+                            <Form.Group controlId='product'>
+                                <Form.Label>Product</Form.Label>
+                                <Form.Control type='product' placeholder="Product">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+
+                    <Row>
+                        <Col>
+                            <Form.Group controlId='routeOfAdmin'>
+                                <Form.Label>Route Of Admin</Form.Label>
+                                <Form.Control type='routeOfAdmin' placeholder="Route Of Admin">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                        <Col>
+                            <Form.Group controlId='siteOfAdmin'>
+                                <Form.Label>Site Of Admin</Form.Label>
+                                <Form.Control type='siteOfAdmin' placeholder="Site Of Admin">
+                                </Form.Control>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+
+                    <Row>
+                        <Col>
+                        </Col>
+                        <Col lg={7}>
+                            <Button variant='primary' type='submit'>Submit</Button>
```

Comment:
Hook up onSubmit action

Reply:
I have onsubmit action on top where the form starts.
![image](https://user-images.githubusercontent.com/22796013/80436586-7cb2fe80-88cd-11ea-8648-15cbf290d5c2.png)

Reply:
I can handle the submit